src/Graphene.API/Controllers/GraphenePersistController.cs

```diff
@@ -35,6 +35,7 @@ public void Post([FromBody] TrackerData trackerData)
             {
                 trackerData.OverrideMinResolution(Resolution.NA);
                 trackerData.Measurement.CoveredResolutions = new List<Resolution>();
+                trackerData.TimeSlot = trackerData.TimeSlot.ToUniversalTime();

```

Comment:
We can't override the TimeSlot in the API. The timeslot is set based on the resolution etc. and shouldn't be overridden.